src/include/esp-idf/bindings.h

```diff
@@ -508,4 +508,9 @@
 #endif // (ESP_IDF_VERSION_MAJOR >= 5 && ESP_IDF_VERSION_MINOR >= 3
 #endif // ESP_IDF_COMP_LCD_ENABLED
 
+// usb serial support
+#if ESP_IDF_VERSION_MAJOR > 4 && defined(SOC_USB_SERIAL_JTAG_SUPPORTED)
+#include "driver/usb_serial_jtag.h"
```

Comment:
Since we already know that this is refactored out from driver/ to its own component in esp-idf v5.3 could you please make it such that it works for v5.0-5.2 this way and includes also correctly for >= 5.3? Thanks!

Reply:
e.g i think the header was renamed right?

Reply:
The location of the header file differs, but in all versions from v4.3 to 5.3 the file is included in C sources ad `#include "driver/usb_serial_jtag.h"`, so it should work for all version no mater the actual location of the file. CI checks are passing, I tested with esp-idf v5.3-beta2 and v5.2.2 and it works just fine

Reply:
ok thanks for checking! If its also present in idf v4.3 than you can remove the ESP_IDF_VERSION_MAJOR check as not needed.

Reply:
i was just assuming that it was a more newer addition but it make sense since esp32c3 support was also there at that time.

Reply:
Removed the version check. I guess we can wait for CI to pass and merge it?